core/camel-support/src/main/java/org/apache/camel/converter/stream/CipherPair.java

```diff
@@ -71,7 +71,7 @@ public Cipher createDecryptor() {
             return deccipher;
         } catch (GeneralSecurityException e) {
             // should not happen
-            throw new IllegalStateException("Could not instantiate decryptor, e");
+            throw new IllegalStateException("Could not instanciate decryptor, e");
```

Comment:
throw new IllegalStateException("Could not instanciate decryptor, e");

->

throw new IllegalStateException("Could not instanciate decryptor", e);

Reply:
Hi, sorry I was blind. I corrected this in a separate commit.